app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/ScheduleData.kt

```diff
@@ -9,4 +9,13 @@ data class ScheduleData(
 
     val roomNames: List<String>
         get() = roomDataList.map { it.roomName }
+
+    val allLectures: List<Lecture>
+        get() = roomDataList.flatMap { it.lectures }.sortedBy { it.dateUTC }
+
+    fun findLecture(lectureId: String): Lecture? {
```

Comment:
That feels a bit extreme. The method is 3 lines long and doesn't involve complex logic. A test class with meaningful coverage would probably be 300+ lines. And it would really only be useful if you expect the implementation of this method to change.

I can replace the method with the following if that makes it easier to understand.

```kotlin
for (roomData in roomDataList) {
    for (lecture in roomData.lectures) {
        if (lecture.lectureId == lectureId) return lecture
    }
}
return null
```

Reply:
Okay. We can go without tests here.